.circleci/config.yml

```diff
@@ -114,11 +114,17 @@ jobs:
       - checkout
       - attach_workspace:
           at: /rest
+      - run:
+          command: |
+            lower=PYPI_TOKEN_GCLOUD_AIO_<<parameters.cwd>>
```

Comment:
Needs to be different tokens for both `aio` and `rest` as well -- `basedir == /rest` for `gcloud-rest-*` tokens and `basedir == /root/project` for `gcloud-aio-*` ones.

Reply:
Yeah I just noticed that there's also rest 🤦

Reply:
Added code to contemplate this